docs/configuration/index.md

```diff
@@ -61,20 +65,13 @@ $ pipx run build --wheel -Ccmake.verbose=true -Clogging.level=INFO
 
 
 ```yaml
-SKBUILD_CMAKE_VERBOSE: true
+SKBUILD_BUILD_VERBOSE: true
 SKBUILD_LOGGING_LEVEL: "INFO"
 ```
 
 
 ````
 
-:::{note}
-
-When using `pip`, make sure you include at least a `-v` argument so that the
-verbosity settings above are displayed.
-
-:::
-
```

Comment:
Why is this note gone? `pip` and `uv pip` hide all output by default. `build` and `uv build` do not.

Reply:
I moved it to the top and made it more generic, didn't want to duplicate it.

Reply:
Ahh, it was rewritten at the top, didn't notice. Okay. I think the note helped it be a bit more called out, but that's fine.